### PR TITLE
adds ability to print a list of guessed letters or words

### PR DESCRIPTION
**Pull request by NicoleCarpenter**

### CODE REVIEW

**Comment by zdavy** [comment]
:+1: except there are a few indentation issues. I'm curious to hear your own opinions on the Guess abstraction.

**Comment by NicoleCarpenter** [comment]
Cool. I thought we fixed the alignment problem over lunch but I guess it did not stick. 

Guess abstraction is great. I was considering that a while ago, but I guess I still had a difficult time deciding on responsibilities. I could figure out something that involved only the guess, like a check on itself, but when I was pulling another parameter in, like `word` to compare, I had a difficult time deciding whether at that point I was "managing the guess" or having the guess reveal something about itself. 

Syntactically, yes, this does read much better, and when I think of the way things are called after the fact (ie: `guess.isValid`, `guess.isFullWord`), it makes sense to have them in their own class. Something like `guess.isCorrectlyGuessedFullWord`, on the other hand, while it reads cleanly, logically my head wants to put it in the other class. I get why it is in guess class now, but it is still more abstract than my brain is used to handling.

File: Sources/Application/extensions/Guess+Equatable.swift [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extension Guess: Equatable {
+}
+
+public func ==(l: Guess, r: Guess) -> Bool {
+	return l.currentGuess == r.currentGuess
+}
```
Comment (zdavy) [inline Sources/Application/extensions/Guess+Equatable.swift]: I was thinking, if there's no reason for the Guess to be equatable in the game, you could move this protocol extension into the Application's test package.

Comment (NicoleCarpenter) [inline Sources/Application/extensions/Guess+Equatable.swift]: That makes sense. I was not really sure at what level that should exist.

**Comment by mnewsome** [comment]
:+1:

**Pull request merged**